src/commands/test.js

```diff
@@ -90,3 +154,122 @@ function getPlatform(logs) {
 
   throw new Error(`Unsupported platform: ${type} ${arch} ${majorVersion}`)
 }
+
+async function runDocker(datasource, opts) {
+  let coverageOpt = opts.get("coverage")
+  let forceOpt = opts.get("force")
+  let versionOpt = opts.get("version")
+  let latestVersion = opts.get("latestVersion")
+  let recompileOpt = opts.get("recompile")
+
+  // Remove binary-install-raw binaries, because docker has permission issues
+  // when building the docker images
+  await filesystem.remove("./node_modules/binary-install-raw/bin")
+
+  // Get current working directory
+  let current_folder = await filesystem.cwd()
+
+  // Build the Dockerfile location. Defaults to ./tests/.docker if
+  // a custom testsFolder is not delcared in the subgraph.yaml
+  let dockerDir = ""
+
+  try {
+    let doc = await yaml.load(filesystem.read('subgraph.yaml', 'utf8'))
+    testsFolder = doc.testsFolder || './tests'
+    dockerDir = testsFolder.endsWith('/') ? testsFolder + '.docker' : testsFolder + '/.docker'
+  } catch (error) {
+    print.error(error)
```

Comment:
Over the `graph-cli`'s codebase we usually print only the `message`, but perhaps this is more useful since it will show the stacktrace as well.

Reply:
Will change it to `error.message` otherwise `print.error()` will print everything